### ISSUE TITLE
It is currently under development?

### ISSUE THREAD

**Issue by tamascsaba** [TheLarkInn/ngc-loader#1]
It is a working loader or under development?

**Comment by yasselavila** [comment]
@TheLarkInn ?

**Comment by TheLarkInn** [comment]
This is severely WIP. A lot of pieces from ng2 and Typescript need to come together before I can start writing this again.

**Comment by yasselavila** [comment]
That's what we wanted to know, if you will continue developing this.... ;-) thanks a lot.

**Comment by TheLarkInn** [comment]
Well I will create a design spec as this will get directly implemented into the angular-cli, but it could be a ways away. Right now the story is pretty crummy for ngc with webpack.

**Comment by yasselavila** [comment]
Yes, it would be nice to have this working not only for angular-cli ... I hope it could be possible.

**Comment by dharapvj** [comment]
Now that angular 2.0 and TS 2.0 is out.. would be you be developing this further?
